pkg/data/set.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package data
```

Comment:
Can we come up with a better naming for the package? I think data is like `utils`/`model` and we are trying to avoid those names. 

What should be inside `data` package? Is it a clear and concise name about what's inside of the package?

Reply:
What do you think about `transformations`? I find it difficult to get a descriptive and concise name. I think this package should be a utility to help transform and manipulate data across different data types. I think that `data type manipulations` would be good, but it doesn't follow the "go best practice" to [name packages](https://go.dev/blog/package-names)